network/address.go

```diff
@@ -653,20 +653,20 @@ func IPv4ToDecimal(ipv4Addr net.IP) (uint32, error) {
 // because it can be used both as an IPv4 or IPv6 endpoint (e.g., in
 // IPv6-only networks).
 func ResolvableHostnames(addrs []Address) []Address {
-	resolveableAddrs := make([]Address, 0, len(addrs))
+	resolvableAddrs := make([]Address, 0, len(addrs))
```

Comment:
This drive-by is in its own commit. View the last commit to see the functional change.